samples/Common/ProductUtilities.cs

```diff
@@ -19,7 +19,7 @@ public static List<Product> GetNewProducts(int num)
                 var product = new Product
                 {
                     ProductID = i,
-                    Cost = 100,
+                    Cost = i,
```

Comment:
do we have to change this as well?

Reply:
not really, but to query I thought it would be better to have different values.

Reply:
Let's at least make the cost `100 + i` or something to clearly differentiate it from the ID